.github/workflows/snyk.yml

```diff
@@ -0,0 +1,25 @@
+# ref: https://github.com/snyk/actions/tree/master/gradle-jdk11
+name: snyk for gradle
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  pull_request_target:
+  
+jobs:
+  security:
+    runs-on: ubuntu-20.04
+    steps:
+    # Set fetch-depth: 0 to fetch commit history and tags for use in version calculation
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Run Snyk to check for vulnerabilities
+      uses: snyk/actions/gradle-jdk11@0.3.0
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
Can you move this form env to `with` as this runs as part of `pull_request_target`?

Reply:
Nope. Doesn't work in that way.